tests/src/rules/prefer-default-export.js

```diff
@@ -129,4 +130,62 @@ ruleTester.run('prefer-default-export', rule, {
       }],
     }),
   ],
-})
+});
+
+context('Typescript', function() {
+  // Typescript
+  const parsers = ['typescript-eslint-parser', 'babel-eslint'];
+
+  if (semver.satisfies(eslintPkg.version, '>5.0.0')) {
+    parsers.push('@typescript-eslint/parser');
```

Comment:
eslint 6 now requires absolute paths here
```suggestion
    parsers.push(require.resolve('@typescript-eslint/parser'));
```